samples/highcharts/data-tools/csv-modifiers/demo.js

```diff
@@ -39,22 +39,6 @@ Dashboards.board('container', {
             options: {
                 csv: csvData,
                 firstColumnsAsNames: true,
```

Comment:
Column assignments must be updated as well, as the named aliases do not exist anymore.

Reply:
There is separate ticked covering this demo: #21373